code/_onclick/click_ctrl.dm

```diff
@@ -0,0 +1,123 @@
+/**
+ * Ctrl click
+ */
+/mob/proc/CtrlClickOn(atom/A)
+	base_click_ctrl(A)
```

Comment:
do we need to have procs like this anymore?
basically, same as `CtrlShiftClick` and even `CtrlMiddleClickOn`
could you call the base directly?

Reply:
Sadly yes :(. This proc is actually overriden by mob subtypes like AI and borgs which don't call base_click_ctrl() but does their own thing.

As for CtrlMiddleClickOn() it first attempts to tag a datum and if not then delegates to base_click_ctrl(). 

CtrlShiftClick() is is also overriden by mob subtypes like AI and borgs which don't call base_click_shift_ctrl() but does their own thing.